jira/client.py

```diff
@@ -96,6 +96,17 @@ def wrapper(*args, **kwargs):
 
     return wrapper
 
+
+def _get_template_list(data):
```

Comment:
Personally I would consider yielding templates instead of returning a list here. Users of the method could then call `list()` on the method result if we do not provide a `_get_template_list` method.